strictTsConfig.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Should we also introduce `noImplicitAny` while we are at it?

Reply:
using `strict` does that.